### ISSUE TITLE
nose2pytest 1.10.0 version not updated in pypi. only 1.0.8 is available

### ISSUE THREAD

**Issue by krishnamohanelluru** [pytest-dev/nose2pytest#27]
(ENV_PY3) test@machine:repo$ pip install nose2pytest==1.10.0
ERROR: Could not find a version that satisfies the requirement nose2pytest==1.10.0 (from versions: 1.0.0, 1.0.1, 1.0.5, 1.0.8)
ERROR: No matching distribution found for nose2pytest==1.10.0

**Comment by schollii** [comment]
Indeed https://pypi.org/project/nose2pytest/ shows 1.0.8.

@nicoddemus any idea why this could be, I'm not sufficiently familiar with the pytest plugins release process

**Comment by nicoddemus** [comment]
Has anyone done a release? It is very simple but not automatic, the instructions are listed here: https://github.com/pytest-dev/nose2pytest/blob/master/RELEASING.rst

(small correction, this is not a pytest plugin, but a command-line tool related to pytest)

**Comment by schollii** [comment]
@nicoddemus thanks, looked at that doc, seems like all the steps were done except new tag. I checked the `deploy` action and it seems that I can just create a tag even without a PR, it will publish to pypi. I will tag 1.0.9 too, based on the github release.

**Comment by nicoddemus** [comment]
Yes, the workflow triggers when a tag is pushed: https://github.com/pytest-dev/nose2pytest/blob/2d560b69a3d329e00664dfb18541455fba82ec72/.github/workflows/deploy.yml#L3-L6

(note the tag should be named `v1.10.0`)

**Comment by nicoddemus** [comment]
Seems like this has been fixed? Closing. If not we can reopen.
